evi-modal-clm/README.md

```diff
@@ -0,0 +1,86 @@
+# EVI ELIZA on Modal
+
+This project sets up a WebSocket server on [Modal](https://modal.com/) for the AI Assistant API, enabling real-time interactions with the EVI agent using the [ELIZA](https://en.wikipedia.org/wiki/ELIZA) chatbot model for human-like conversations.
```

Comment:
Maybe use the terminology "custom language model" so people can connect it back to our docs?